.travis.yml

```diff
@@ -12,5 +12,17 @@ php:
   - 7.1
   - hhvm
 
+matrix:
+  include:
+    - php: 7
+      env: PHPSTAN=1
+    - php: 7.1
+      env: PHPSTAN=1
```

Comment:
Of course PHPUnit does not need to run in this job, there'd need to be another flag for turning PHPUnit off.